week-4/Homework/mandatory/4-groceries.js

```diff
@@ -5,8 +5,23 @@
 
 let groceriesToBuy = [];
 
+
+
 let groceryList = {
   item1: "",
   item2: "",
   item3: ""
 };
+
+groceryList.item1 = 'Potatoes';
+groceryList.item2 = 'Orange Juice';
+groceryList.item3 = 'Rice';
+
+
+const objEntries = Object.values(groceryList);
```

Comment:
I think it does need to use foreach loop because : Object.values -> return array